unitTests/controllers/FacetSliderQueryControllerTest.ts

```diff
@@ -122,6 +122,17 @@ export function FacetSliderQueryControllerTest() {
         requestForFullRange = builder.groupByRequests[controller.lastGroupByRequestForFullRangeIndex];
         expect(requestForFullRange.constantQueryOverride).toContain('@foo>1970');
       });
+
+      it('should contain a special expression to filter out invalid document if the field is a date for the graph request', () => {
```

Comment:
I do not understand this test.

- Does the special expression refer to `'@foo>1970'` or to `something`?

- What is an invalid document in this context? One that does not have the `@foo` field? Or that has the value to less than 1970?

- Are two expectations needed for this one test, or can it be broken up into two tests with separate descriptions?

- Should there be a test/expectation for the value of advancedQueryOverride?